examples/texteditor/src/main/java/com/example/texteditor/EditorViewModel.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.texteditor
+
+import android.content.Context
+import androidx.databinding.ObservableField
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dev.yorkie.core.Client
+import dev.yorkie.document.Document
+import dev.yorkie.document.json.JsonText
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.NonCancellable
+import kotlinx.coroutines.launch
+import kotlin.math.max
+
+class EditorViewModel : ViewModel(), YorkieEditText.TextEventHandler {
+    private lateinit var client: Client
+    private val document = Document(Document.Key(DOCUMENT_KEY))
+    val content = ObservableField("")
```

Comment:
StateFlow is more recommended (https://developer.android.com/topic/libraries/data-binding/observability#stateflow)